source/WebConfig/webconfig_internal.c

```diff
@@ -420,13 +412,15 @@ int handleHttpResponse(long response_code, char *webConfigData, int retry_count,
 	else if(response_code == 204)
 	{
 		WebConfigLog("No configuration available for this device. response_code:%d\n", response_code);
+		addWebConfigNotifyMsg(configURL, response_code, NULL, 0, RequestTimeStamp , "v1.0-NONE");
```

Comment:
Remove v1.0, its only NONE.
Schema version is separate.

Reply:
Incorporated.